test/api/unit/services/SQS.test.js

```diff
@@ -80,8 +80,6 @@ describe('SQS', () => {
     context("building a site's default branch", () => {
       it('should set an empty string for BASEURL in the message for a site with a custom domain', (done) => {
         const domains = [
-          'www.example.com',
```

Comment:
Are we sure? This is what I'm looking at:

```javascript
> validator.isURL('www.example.com')
true
```

So I'd think a domain without a protocol could still make it's way into database.